code/modules/mob/living/carbon/human/species/station/traits_vr/negative.dm

```diff
@@ -133,3 +133,18 @@
 /datum/trait/colorblind/para_taj/apply(var/datum/species/S,var/mob/living/carbon/human/H)
 	..(S,H)
 	H.add_modifier(/datum/modifier/trait/colorblind_taj)
+
+/datum/trait/coldprone
+	name = "Cold prone"
+	desc = "You are much more prone to colder temperatures than other species, and become uncomfortable much more easily in cold environments."
+	cost = -2
```

Comment:
I feel like this may be a little too many points for such a niche weakness. -1 would feel a bit more in line with other nicher weaknesses, IMO.

Reply:
Especially when these weaknesses can be entirely ignored by wearing a suit, like everyone already does in these environments.

Reply:
If it's gonna be -1, though, I feel like it should like that across the board. I feel like even though it's niche, -2 accounts for the fact that it's such a big change in terms of actual temperature levels. I don't feel like it's appropriate for it to cost 2 points for someone to add 120 degrees celcius to their total range of survivable temperatures. That's kinda borked in my opinion.

Reply:
> Especially when these weaknesses can be entirely ignored by wearing a suit, like everyone already does in these environments.

Okay that's actually a good point. What do you guys think of 1 point for both hot/cold prone and resistance?

Reply:
The cold resistance one could be two since it makes you safe in the temperatures outside on 3b, but it's pretty rare that somewhere is heated up by *only* 120 degrees, since that's almost always caused by fires that bump things up by thousands of degrees.

Reply:
3b requires some form of suiting up regardless on account of toxicity, so I doubt it changes much event there.

Reply:
Yeah, honestly that's what I was thinking. It's pretty niche both ways.

Reply:
I disagree with anything being 2 points. There is almost no reason you would ever find yourself in the cold if you purposely avoided it.

Reply:
> I disagree with anything being 2 points. There is almost no reason you would ever find yourself in the cold if you purposely avoided it.

Hence I have changed them to all be 1 point!

Reply:
Generally speaking when it comes to environmental resistance in SS13, it can be summed up as "you're immune, or you're not". Having a partial resist barely makes a difference over having no resistance, you still can't hang around. Being slightly more vulnerable to the cold just means you have the exact same set of environments you can't survive in. Hence why you wear a radsuit and not an engineering hardsuit to reset the gravity generator - sure, you have 85% rad resistance in the hardsuit, but it chucks out so many rads that even 15% of an assload of rads is gonna suck, and you have complete immunity available in the locker by the door.

Taking 2 damage a second instead of 1 damage a second is barely a disability when both of you are going to put on a suit and take 0 damage a second.

Reply:
> Generally speaking when it comes to environmental resistance in SS13, it can be summed up as "you're immune, or you're not". Having a partial resist barely makes a difference over having no resistance, you still can't hang around. Being slightly more vulnerable to the cold just means you have the exact same set of environments you can't survive in. Hence why you wear a radsuit and not an engineering hardsuit to reset the gravity generator - sure, you have 85% rad resistance in the hardsuit, but it chucks out so many rads that even 15% of an assload of rads is gonna suck, and you have complete immunity available in the locker by the door.
> 
> Taking 2 damage a second instead of 1 damage a second is barely a disability when both of you are going to put on a suit and take 0 damage a second.

So! This is actually not what these changes do. We're shifting the point at which someone starts to take damage around rather than messing with how much damage they take. For example, with heat prone, setting the thermostat past ~33C will cause you to start taking burn damage! Same thing with Cold prone and setting it to 0C

Reply:
I think what they (Scree) means is that usually when such things matter, it's too extreme to really make a difference. A fire is going to put the temperature into the thousands really quickly, and superheat the air. A couple degrees extra heat resistance isn't going to help. Simply setting the thermostat up to 33C or etc isn't going to... really do much.

It's a matter of the environment in which temperature matters is already so extreme that small changes like this wouldn't matter much, you can start taking damage at 30C or 35C or 40C, but when the room is 200+C, it won't help you there. Likewise for Virgo's external freezing temperatures.

Reply:
Basically whay Rykka said. Unless it's going to make you start taking damage in normal station temperatures and require you to wear an environment suit, it's not really a disadvantage in any situation that won't be a disadvantage for anyone else.

If the external atmosphere wasn't deadly poison then having a perk to survive in _those_ temperatures would be neat and a definite advantage. Maybe something for if we get a breathable-atmosphere snow map.

Reply:
What Rykka says is a good point. I could be misunderstanding, but it seems these are pretty minor weaknesses/strengths. I think there _is_ a place for  extreme temperatures being livable (See: Rockmen from FTL), but I'm dubious of the use and implementation. As far as the _minor_ adjustments, my sentiments mirror that of Rykka and Scree. I struggle to think of a situation where you would be in a room with a temperature that is greater than comfort but lower than **_oh god oh fuck_**.

Reply:
> Basically whay Rykka said. Unless it's going to make you start taking damage in normal station temperatures and require you to wear an environment suit, it's not really a disadvantage in any situation that won't be a disadvantage for anyone else.
> 
> If the external atmosphere wasn't deadly poison then having a perk to survive in _those_ temperatures would be neat and a definite advantage. Maybe something for if we get a breathable-atmosphere snow map.

We actually do have two of these in the gateway.

I imagine one of the niche (but still plausible) uses being, rather than having to switch into a winter coat to go outside, you could just slap on a gas mask and call it a day with whatever suit you were already wearing, potentially saving some time and also being more convenient.

Insofar as the weaknesses go, like I mentioned, there's the snowy gateway sites. Also, it's happened to me in the past where a small phoron fire or explosion from something like toxins would raise the temperature >40c but not to dangerous amounts for most species, so another niche situation where it would be like ohfuckthatonethingactuallyisaweakness.

Which is why I think they're valid as -1 negative traits. They're largely ignorable but they do nonetheless have those situations where they could end up applying.